test-loop-tests/src/examples/resharding.rs

```diff
@@ -7,21 +7,23 @@ use near_o11y::testonly::init_test_logger;
 use near_primitives::epoch_manager::EpochConfigStore;
 use near_primitives::shard_layout::ShardLayout;
 use near_primitives::types::AccountId;
-use near_primitives::version::ProtocolFeature;
+use near_primitives::version::PROTOCOL_VERSION;
 
 use crate::setup::builder::TestLoopBuilder;
 use crate::utils::setups::derive_new_epoch_config_from_boundary;
 
 #[test]
-fn test_resharding() {
+fn resharding_example_test() {
     init_test_logger();
 
-    let base_shard_layout = ShardLayout::multi_shard(3, 3);
+    // Note: having base layout version set to 3 is important
```

Comment:
If you know would be good to mention why is it important.

Reply:
I have no idea why, but it doesn't work otherwise 🤷

Reply:
Lol, I had a similar question in mind, why exactly is the version 3 important. But anyway, good to mention it here.